exercises/parallel-letter-frequency/package.yaml

```diff
@@ -1,5 +1,7 @@
 name: parallel-letter-frequency
-version: 0.1.0.2
+version: 0.1.0.3
+
+ghc-options: -threaded -with-rtsopts=-N -O2
```

Comment:
This lines causes the following warnings:

```
Configuring parallel-letter-frequency-0.1.0.3...
Warning: 'ghc-options: -threaded' has no effect for libraries. It should only be used for executables.
Warning: 'ghc-options: -with-rtsopts' has no effect for libraries. It should only be used for executables.
```

Besides the warnings, the only real side effect of this is enabling `-threaded -with-rtsopts=-N -O2` for the test suite, which doesn't need it, but the slower compilation from `-O2` probably has an insignificant effect in the overall building time.